app/views/orders/show.html.erb

```diff
@@ -15,7 +15,7 @@
     <tfoot>
       <tr>
         <td colspan="4">Total:</td>
-        <td colspan="2"><%= @total %></td>
+        <td colspan="2" class="subtracted"><%= @total %></td>
```

Comment:
why call it like that? since there is only one total on the page, you can give it an id `total`, it would be clearer